src/schemas-tests/pa-yaml/v3.0/Examples/Src/Screens/FormsScreen2.pa.yaml

```diff
@@ -65,6 +66,7 @@ Screens:
                             Children:
                               - DataCardKey7:
                                   Control: Label
+                                  IsLocked: true
```

Comment:
only the parent data card should get the IsLocked property set.
the code at design time in the studio applies the final IsLocked value by looking whether it has a parent that is locked.

make sure our test code keeps this semantic.